tests/integration/test_logtest/test_remove_session/data/remove_session.yaml

```diff
@@ -0,0 +1,65 @@
+---
+-
+  name: "Remove invalid token type"
+  description: "Check remove session"
+  test_case:
+  -
+    input: '{"remove_session": 6}'
+    output: "{\"messages\":[\"ERROR: (7316): Failure to remove session. remove_session JSON field must be a string\"],\"codemsg\":-2}"
+    stage: 'Remove invalid token type'
+-
+  name: "Remove invalid json"
+  description: "Check remove session"
+  test_case:
+  -
+    input: '{"remove_session": asd}'
+    output: "{\"messages\":[\"ERROR: (7306): Error parsing JSON\",\"ERROR: (7307): Error in position 19, ... session\\\": asd} ...\"],\"codemsg\":-2}"
+    stage: 'Remove invalid json'
+-
+  name: "Remove null token"
```

Comment:
I think "Remove invalid empty json object token" define better the test case ( based on JSON Data types)